src/main/io/gps.h

```diff
@@ -67,23 +67,11 @@ typedef enum {
     CFG_MOT_GNSSSPEED_THRS = 0x20250038, // U1
     CFG_MOT_GNSSDIST_THRS = 0x3025003b, // U2
     CFG_ANA_USE_ANA = 0x10230001, // L
-    CFG_MSGOUT_NMEA_ID_VTG_I2C = 0x209100b0, // U1
-    CFG_MSGOUT_NMEA_ID_VTG_SPI = 0x209100b4, // U1
     CFG_MSGOUT_NMEA_ID_VTG_UART1 = 0x209100b1, // U1
```

Comment:
This other "NMEA" constants are not for NMEA?

Reply:
Yes, these should stay